README.md

```diff
@@ -103,10 +105,11 @@ Please file bugs as you see them! See [CONTRIBUTING.md](CONTRIBUTING.md) for mor
         version.
   - [ ] Adds additional imports when necessary.
   - [ ] Warns on other major breaking changes to elements.
-  - [ ] Upgrades mixins from 0.5 into behaviors of 1.0.
+  - [x] Upgrades mixins from 0.5 into behaviors of 1.0.
```

Comment:
could use some details here - does this upgrade the mixin application, declaration, or both?